apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js

```diff
@@ -68,11 +70,138 @@ const logger = getLogger('sbbol/SbbolFintechApi')
+ * @property {string} debitBlockedBeginDate (string, optional)    Дата начала действия ограничения
+ * @property {string} debitBlockedCause (string, optional)    Основание ареста
+ * @property {string} debitBlockedEndDate (string, optional)    Дата снятия ограничения
+ * @property {string} debitBlockedInitiator (string, optional)    Наименование органа, наложившего арест
+ * @property {string} debitBlockedTaxAuthorityCode (string, optional)    Код налогового органа, наложившего арест
+ * @property {number} minBalance (number, optional)    Минимальный поддерживаемый (неснижаемый) остаток на счете
+ * @property {string} mode (string, optional)    Режим работы счета = [STANDART, FORBIDDEN_RECEIVING, ONLY_RECEIVING]
+ * @property {string} name (string, optional)    Наименование счета
+ * @property {boolean} notDelay (boolean, optional)    Признак возможности проведения неотложных платежей
+ * @property {string} number (string, optional)    Номер счета (20 сиволов)
+ * @property {string} openDate (string, optional)    Дата открытия счета
+ * @property {number} overdraft (number, optional)    Сумма общего лимита овердрафта в валюте счета
+ * @property {boolean} passive (boolean, optional)    Признак пассивности счета
+ * @property {string} state (string, optional)    Состояние счета = [OPEN, BLOCKED, CLOSED]
+ * @property {string} type (string, optional)    Тип счета = [assuranceRegistration, calculated, transit, specialTransit, budget, loan, deposit]
+ * @property {boolean} urgent (boolean, optional)    Признак возможности проведения срочных платежей
+ */
+
+/**
+ * @typedef SbbolClientInfo
+ * @property {string} shortName (string, optional)	Сокращенное наименование организации
+ * @property {string} fullName (string, optional)	Полное наименование организации
+ * @property {string} ogrn (string, optional)	ОГРН
+ * @property {string} inn (string, optional)	ИНН / КИО
+ * @property {string} orgForm (string, optional)	Организационно-правовая форма организации клиента
+ * @property {Address[]} addresses (Address[], optional)	Адреса (все заведенные для клиента)
+ * @property {SbbolAccount[]} accounts (SbbolAccount[], optional)	Счета организации клиента, доступные партнеру
+ * (если клиент выбрал счет при заключении оферты)
+ * @property {string[]} kpps (string[], optional)	КПП (все заведенные для организации клиента)
+ * @description There are more fields that can only be obtained by expanding the request scope,
+ * so for a complete list of what you can get, you need to go to the SberBusiness documentation
+ */
+
+/**
+ * SBBOL format response parser
+ *
+ * @param {{data: Object, statusCode: Number}} response SBBOL formatted response
+ * @param {Number} statusCode successful response code
+ * @param {String} msg successful response code
+ * @return {{{error: any, statusCode: Number}}|{data: any}}
+ */
+function parseSbbolResponse (response, statusCode, msg) {
```

Comment:
I don't see reusing of `msg` function argument.
It's better to write "Parsing error" message right in this function.

```suggestion
function parseSbbolResponse (response, statusCode) {
```

Reply:
```
    } catch (error) {
        return logger.error({ msg, error })
    }
```